Python/optimizer.c

```diff
@@ -1245,8 +1248,18 @@ uop_optimize(
     }
     assert(length < UOP_MAX_TRACE_LENGTH);
     OPT_STAT_INC(traces_created);
-    char *env_var = Py_GETENV("PYTHON_UOPS_OPTIMIZE");
-    if (env_var == NULL || *env_var == '\0' || *env_var > '0') {
+
+    if (!uop_optimize_initialized) {
+        char *env_var = Py_GETENV("PYTHON_UOPS_OPTIMIZE");
+        if (env_var == NULL || *env_var == '\0' || *env_var > '0') {
+            uop_optimize_flag = 1;
+        } else {
```

Comment:
Please make sure all new code is formatted according to PEP-7.

```suggestion
        }
        else {
```